packages/mdc-list/mdc-list.scss

```diff
@@ -105,6 +105,11 @@
   @include mdc-typography-overflow-ellipsis;
 }
 
+// Disable interaction on label elements that may automatically toggle correspoding checkbox / radio input.
+label.mdc-list-item__text[for] {
```

Comment:
Since we're including the class and the attribute selector, would this selector work just as well without `label`? (Since `for` pretty much scopes this down to `label` already)